wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -56,34 +56,43 @@ public final class LmiConfigRecommender {
 
     private LmiConfigRecommender() {}
 
-    static void configure(Properties lmiProperties, LmiUtils.HuggingFaceModelConfig modelConfig) {
-        setRollingBatch(lmiProperties, modelConfig);
-        setEngine(lmiProperties);
+    static void configure(
+            ModelInfo<?, ?> modelInfo,
+            Properties lmiProperties,
+            LmiUtils.HuggingFaceModelConfig modelConfig) {
+        String features = Utils.getEnvOrSystemProperty("SERVING_FEATURES");
+        setDynamicBatch(lmiProperties, modelConfig, modelInfo, features);
+        setRollingBatch(lmiProperties, modelConfig, features);
+        setEngine(lmiProperties, modelConfig, features);
         setTensorParallelDegree(lmiProperties);
     }
 
     private static void setRollingBatch(
-            Properties lmiProperties, LmiUtils.HuggingFaceModelConfig modelConfig) {
+            Properties lmiProperties,
+            LmiUtils.HuggingFaceModelConfig modelConfig,
+            String features) {
         // If dynamic batch is enabled, we don't enable rolling batch.
         if (Integer.parseInt(lmiProperties.getProperty("batch_size", "1")) > 1) {
             return;
```

Comment:
In setEngine, we are seeing if rolling_batch==disable, do we need to set the value here explicitly to disable? I think i would prefer that well, since we may change the default in the handlers from disable->auto at some point

Reply:
Yeah, Right now, in this changes, for t5, we ask the user to explicitly set the rolling batch to disable. So I guess, the UX would be like if the user specifies batch_size then we disable rolling batch automatically. If they don't we try to do rolling batch based on config type. 

But for t5, we enable dynamic batch automatically. In that case, we need to disable rolling batch here?

Reply:
In our properties manager, we currently default to disabled rolling batch https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/properties_manager/properties.py#L49.

We have been discussing making the default "auto", so rolling batch is enabled by default. If that were the case, would the logic here still work? For T5, if user doesn't explicitly disable rolling batch, but sets a dynamic batch size, is there a chance both could be enabled? Do we explicitly set rolling_batch=disable somewhere when it's t5 trtllm?

I don't expect us to update this default behavior for this release, so it's not a big deal now. But if we update the default rolling batch value to auto before trt enables rolling batch for t5, could that cause issues down the line?

Reply:
As discussed offline, for JIT (LCNC) experience, automatically disabling rolling batch if its t5 and also setting the entrypoint